engine/helper/pka_helper.c

```diff
@@ -86,6 +173,119 @@ static uint32_t gbl_engine_finish;
         PKA_PRINT(PKA_ENGINE, fmt_and_args);    \
 })
 
+ENGINE_PKA_KEYPAIR *engine_pka_keypair_new(int nid, int flag, int size)
+{
+    ENGINE_PKA_KEYPAIR *kpair = NULL;
+
+    kpair = OPENSSL_secure_malloc(sizeof(*kpair));
+    if (kpair == NULL)
+    {
+        printf("ERROR: ENGINE_PKA_KEYPAIR: Memory allocation failed\n");
```

Comment:
Q: Why we are using ENGINE_PKA_KEYPAIR?

This implies that the error is returned by a different module, perhaps, we should use the function name?

Reply:
Please see my above comment regarding error print format.